plugins/kotlin-as-java/src/main/kotlin/OriginalProperty.kt

```diff
@@ -0,0 +1,11 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.model.DFunction
+import org.jetbrains.dokka.model.DProperty
+import org.jetbrains.dokka.model.properties.ExtraProperty
+import org.jetbrains.dokka.model.properties.MergeStrategy
+
+class OriginalProperty(val original: DProperty) : ExtraProperty<DFunction> {
+    override val key = OriginalProperty
+    companion object: ExtraProperty.Key<DFunction, OriginalProperty>
+}
```

Comment:
Out of context, a bit unclear what `Original` is exactly. Original in relation to what?

After seeing the usage, it starts to make sense, but maybe it's worth adding a simple KDoc like `original function property before any merging or modificatoin took place` to save some brain fuel

Also, I didn't find any actual usages of this - is it something you'll need in the future or use in your plugins?